channels/ru.m3u

```diff
@@ -66,8 +66,6 @@ https://strm.yandex.ru/kal/hdmedia/hdmedia0.m3u8
 https://sc.id-tv.kz/hdl.m3u8
 #EXTINF:-1 tvg-id="HITV.ru" tvg-name="HITV" tvg-country="RU" tvg-language="Russian" tvg-logo="" group-title="Music",HITV (1080p)
 https://strm.yandex.ru/kal/hittv/hittv0.m3u8
-#EXTINF:-1 tvg-id="KrikTV.ru" tvg-name="Krik TV" tvg-country="RU" tvg-language="" tvg-logo="" group-title="",Krik TV (720p)
-http://cdn-01.bonus-tv.ru:8080/kriktv_edge/index.m3u8
```

Comment:
Russian peepz, do you have something on https://strm.yandex.ru/kal/kriktv/kriktv0.m3u8 ? Might be geolocked if works for you.